biomaj/bank.py

```diff
@@ -525,10 +527,36 @@ def save_session(self):
 
         self.bank = self.banks.find_one({'name': self.name})
 
+    def format_release(self, release, reverse=False):
+        """
+        Check the release does not contains any dot '.' character that should make MongoDB complain
+
+        :param release: release number to check
+        :type release: str
+        :param reverse: Do the reverse replacement
+        :type reverse: bool
+        :return: Release string with dot '.' replaced with character set in config (release.replace)
+        :rtype: str
+        """
+        if not self.config.get('release.replacedot'):
```

Comment:
Ok, this means that we "hardcode" a character to search in the method itself if not defined in config? Something like:
```
        # default character search and replace 
        char_search = '.'
        char_replace = '#'

        if self.config.get('release.replacedot'):
            char_replace = self.config.get('release.replacedot')

        if reverse:
            switch = char_search
            char_search = char_replace
            char_replace = switch

        if str(release).find(char_search):
            return str(release).replace(char_search, char_replace)
        return str(release)
```
In such case we do not throw any error if `release.replacedot` is not set ?